src/nvim/option.c

```diff
@@ -2666,6 +2667,7 @@ did_set_string_option (
     } else {
       // load or unload key mapping tables
       errmsg = keymap_init();
+      ft_changed = STRCMP(oldval, *varp) != 0;
```

Comment:
This doesn't seem to be in the vim patch, nor does FileType autocmd docs mention being fired when `'keymap'` (or `'syntax'` below) is set. What is the motivation for this?

Reply:
I was trying if it works, I thought it will work I see the errmsg there so, left it, may be I forgot to remove it, no problem, I remove it

Reply:
I was trying if it works, I thought it will work I see the errmsg there so, left it, may be I forgot to remove it, no problem, I remove it